webserver/views/api/v1/core.py

```diff
@@ -91,13 +92,15 @@ def get_high_level(mbid):
 
     :query n: *Optional.* Integer specifying an offset for a document.
     :query map_classes: *Optional.* If set to 'true', map class names to human-readable values
+    TODO: provide a link to what these mappings are
```

Comment:
Should fix this TODO, or is the plan to add a link to readthedocs when you deploy the docs changes?

Reply:
I've opened a ticket at https://tickets.metabrainz.org/browse/AB-444 for this